.github/workflows/pull_request.yml

```diff
@@ -24,6 +24,17 @@ jobs:
             linux_nightly_6_0_arguments_override: "--explicit-target-dependency-import-check error"
             linux_nightly_main_arguments_override: "--explicit-target-dependency-import-check error"
 
+    integration-tests:
+        name: Integration Tests
+        # Workaround https://github.com/nektos/act/issues/1875
+        uses: apple/swift-nio/.github/workflows/swift_matrix.yml@main
```

Comment:
I don't think we should use the matrix here. Testing examples with latest should be fine.

Reply:
What's the benefit of the matrix here? I still don't think we should use it.

Reply:
@fabianfett because that's the only script I found in Swift-NIO.  Not using it would mean write a specific action for this project.  Is this OK for you ?